conf/roslin.config

```diff
@@ -56,6 +52,10 @@ singularity {
     runOptions   = '-p -B "$TMPDIR"'
     enabled      = true
     autoMounts   = true
-    cacheDir     = '/exports/cmvm/eddie/eb/groups/alaw3_eb_singularity_cache'
+    // Define the singularity cache directory depending on the presence of the NFX_SGE_PROJECT variable
+    // User without compute project can't access to the shared cache directory.
+    // So, they need to store singularity images into the work directory.
+    cacheDir     = System.getenv('NFX_SGE_PROJECT') ?
```

Comment:
`libraryDir` instead? (In which case one might not need the conditional?)

Reply:
The cacheDir better fit to our setup. It convenient for us to allow user to add new images when they run a new pipeline / new version. 
Thanks for the suggestion.